ci/tests/rp/test.sh

```diff
@@ -10,13 +10,16 @@ else
 fi
 
 git clone -b v$rp_version --single-branch https://github.com/radical-cybertools/radical.pilot.git
-
 cd radical.pilot
+
 echo '--- smoke test'
-./examples/00_getting_started.py
+./examples/00_getting_started.py local.localhost_test
 ret=$?
 echo "--- smoke test $ret"
 
+SID=$(ls -rt | grep rp.session)
+rm -rf "$HOME/radical.pilot.sandbox/$SID" || true
```

Comment:
Sorry, saw that only after I pressed the button, but why is the `|| true` needed?  The `-f` flag should result in successful run anyway?

Also, it might make sense to check if `SID` is indeed not empty.  Maybe:
```sh
test -z "$SID" || rm -rf "$HOME/radical.pilot.sandbox/$SID"
```
?

Reply:
updated accordingly

p.s. had `|| true` so it wouldn't fail anyhow, but agree to have it as you proposed